common.js

```diff
@@ -69,7 +69,8 @@ function generateFinalBasename (opts) {
 }
 
 function generateFinalPath (opts) {
-  return path.join(opts.out || process.cwd(), generateFinalBasename(opts))
+  if (opts.out) return path.join(opts.out, generateFinalBasename(opts))
```

Comment:
Sorry, I just woke up. I meant to say, to fully fix #385, replace `if (opts.out)` with

```javascript
if (typeof(opts.out) === 'string')
```